src/wp-admin/includes/class-wp-debug-data.php

```diff
@@ -410,552 +252,554 @@ public static function debug_data() {
-					'value' => wp_get_font_dir()['basedir'],
-				),
-				'fonts_size'     => array(
-					'label' => __( 'Fonts directory size' ),
-					'value' => $loading,
-					'debug' => 'loading...',
-				),
-				'database_size'  => array(
-					'label' => __( 'Database size' ),
-					'value' => $loading,
-					'debug' => 'loading...',
-				),
-				'total_size'     => array(
-					'label' => __( 'Total installation size' ),
-					'value' => $loading,
-					'debug' => 'loading...',
-				),
-			);
-		}
+		$info['wp-paths-sizes'] = array(
+			/* translators: Filesystem directory paths and storage sizes. */
+			'label'  => __( 'Directories and Sizes' ),
+			'fields' => $fields,
+		);
+		return $info;
+	}
 
+	/**
+	 * Populate the drop-ins section of the debug data.
+	 *
+	 * @since tbd
+	 *
+	 * @param array $info The debug information to be added to the core information page.
+	 *
+	 * @return array
+	 */
+	public static function wp_dropins( array $info ): array {
 		// Get a list of all drop-in replacements.
 		$dropins = get_dropins();
 
-		// Get dropins descriptions.
+		// Get drop-ins descriptions.
 		$dropin_descriptions = _get_dropins();
 
-		// Spare few function calls.
-		$not_available = __( 'Not available' );
-
+		$fields = array();
 		foreach ( $dropins as $dropin_key => $dropin ) {
-			$info['wp-dropins']['fields'][ sanitize_text_field( $dropin_key ) ] = array(
+			$fields[ sanitize_text_field( $dropin_key ) ] = array(
 				'label' => $dropin_key,
 				'value' => $dropin_descriptions[ $dropin_key ][0],
 				'debug' => 'true',
 			);
 		}
 
-		// Populate the media fields.
-		$info['wp-media']['fields']['image_editor'] = array(
-			'label' => __( 'Active editor' ),
-			'value' => _wp_image_editor_choose(),
+		$info['wp-dropins'] = array(
+			'label'       => __( 'Drop-ins' ),
+			'show_count'  => true,
+			'description' => sprintf(
+			/* translators: %s: wp-content directory name. */
+				__( 'Drop-ins are single files, found in the %s directory, that replace or enhance WordPress features in ways that are not possible for traditional plugins.' ),
+				'<code>' . str_replace( ABSPATH, '', WP_CONTENT_DIR ) . '</code>'
+			),
+			'fields'      => $fields,
 		);
 
-		// Get ImageMagic information, if available.
-		if ( class_exists( 'Imagick' ) ) {
-			// Save the Imagick instance for later use.
-			$imagick             = new Imagick();
-			$imagemagick_version = $imagick->getVersion();
-		} else {
-			$imagemagick_version = __( 'Not available' );
-		}
+		return $info;
+	}
 
-		$info['wp-media']['fields']['imagick_module_version'] = array(
-			'label' => __( 'ImageMagick version number' ),
-			'value' => ( is_array( $imagemagick_version ) ? $imagemagick_version['versionNumber'] : $imagemagick_version ),
-		);
+	/**
+	 * Populate the active theme section of the debug data.
+	 *
+	 * @since tbd
+	 * @global array $_wp_theme_features
+	 *
+	 * @param array $info The debug information to be added to the core information page.
+	 *
+	 * @return array
```

Comment:
```suggestion
	 * @since tbd
	 *
	 * @global array $_wp_theme_features
	 *
	 * @param array $info The debug information to be added to the core information page.
	 * @return array
```

Minor docblock update